src/makeRobot.js

```diff
@@ -39,6 +39,66 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    chip: version,
```

Comment:
```suggestion
    version,
```
Why do you need another property name?

Reply:
more understandible fro me